### PR TITLE
Stubbing out the LiveAuctions UI

### PR DESCRIPTION
**Pull request by orta**
The class names are all just stubs while I work out responsibilities / roles for each bit. As is the UI structure, I want things to work with, not a perfect setup. Will comment inline if anything seems funny. 

screen shot 2016-03-02 at 12 21 23 pm

### CODE REVIEW

File: Artsy/Models/API_Models/LiveAuctionLot.h [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+@property (nonatomic, copy, readonly) NSString *currency;
+@property (nonatomic, copy, readonly) NSString *currencySymbol;
+
+@property (nonatomic, assign, readonly) NSInteger lowEstimateCents;
+@property (nonatomic, assign, readonly) NSInteger highEstimateCents;
+@property (nonatomic, assign, readonly) NSInteger askingPriceCents;
+@property (nonatomic, assign, readonly) NSInteger onlineAskingPriceCents;
+
+@end
```
Comment (orta) [inline Artsy/Models/API_Models/LiveAuctionLot.h]: we have to use a custom JSON parser for this, so the object graph is entirely self-contained here. ( e.g. no `lot.artwork`  ) you can see in the JSON below that there's little details for this stuff.

File: Artsy/Models/API_Models/LiveAuctionLot.m [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
++ (NSDictionary *)JSONKeyPathsByPropertyKey
+{
+    return @{
+        ar_keypath(LiveAuctionLot.new, liveAuctionID) : @"id",
+
+        ar_keypath(LiveAuctionLot.new, artworkTitle) : @"artwork.title",
+        ar_keypath(LiveAuctionLot.new, artistName) : @"artist.name",
+        ar_keypath(LiveAuctionLot.new, imageDictionary) : @"artwork.image"
+    };
```
Comment (orta) [inline Artsy/Models/API_Models/LiveAuctionLot.m]: we only have to define things that cannot be determined via introspection

File: Artsy.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -336,6 +336,8 @@
 		6019A9081C7E76EF00E292FA /* LiveAuctionsAssets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 6019A9071C7E76EF00E292FA /* LiveAuctionsAssets.xcassets */; };
 		6019A90D1C7F9EF600E292FA /* LiveAuctionViewControllerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6019A90C1C7F9EF600E292FA /* LiveAuctionViewControllerTests.swift */; };
 		601C317816582BA30013E061 /* ARRouter.m in Sources */ = {isa = PBXBuildFile; fileRef = 601C317716582BA30013E061 /* ARRouter.m */; };
+		601E31AB1C80A55F00D09F73 /* live_actions.json in Resources */ = {isa = PBXBuildFile; fileRef = 601E31AA1C80A55F00D09F73 /* live_actions.json */; };
```
Comment (sarahscott) [inline Artsy.xcodeproj/project.pbxproj]: Is this supposed to be live_auctions?

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionViewController.swift [hunk 4]
```diff
@@ -62,6 +69,44 @@ class LiveAuctionViewController: UIViewController {
...
+
+        let metadataStack = ORStackView()
+        metadataStack.bottomMarginHeight = 0
+        view.addSubview(metadataStack)
+        metadataStack.constrainTopSpaceToView(artworkSelectionView, predicate: "20")
+        metadataStack.constrainWidthToView(view, predicate: "-40")
+        metadataStack.alignCenterXWithView(view, predicate: "0")
+
+        let artistNameLabel = UILabel().then {
+            $0.text = "Damien Hirst"
```
Comment (sarahscott) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionViewController.swift]: Is this just test data for now?

**Comment by sarahscott** [comment]
Looks good to me! Just confused about live_actions vs live_auctions

File: Artsy.xcodeproj/project.pbxproj (hunk 3 above)
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: hah - yeah, but it's a temp file till we do real networking, it can stay bad

**Pull request merged**